src/code/z_camera.c

```diff
@@ -6,13 +6,30 @@
 
 s16 Camera_RequestSettingImpl(Camera* camera, s16 requestedSetting, s16 flags);
 s32 Camera_RequestModeImpl(Camera* camera, s16 requestedMode, u8 forceModeChange);
-s32 Camera_QRegInit(void);
 s32 Camera_UpdateWater(Camera* camera);
 
+#if OOT_DEBUG
+s32 Camera_QRegInit(void);
+#endif
+
+#if OOT_DEBUG
+#define CAM_DEBUG_RELOAD_PREG(camera) \
+    if (R_RELOAD_CAM_PARAMS)          \
+    Camera_CopyPREGToModeValues(camera)
+#else
```

Comment:
tiny thing but we can probably just include braces in the definition. its not a huge deal since its hidden by the macro, but if we can continue to follow the repos style we might as well imo

Reply:
Alright, I've added braces and a `(void)0`